app/views/layouts/application.html.erb

```diff
@@ -24,34 +24,37 @@
     <%= javascript_importmap_tags %>
   </head>
 
-  <body class="<%= 'body--index' if request.path_info == '/' %>" data-controller="nav">
+  <body class="<%= 'body--index' if request.path_info == '/' %>" data-controller="nav" data-nav-expanded-class="mobile-nav-is-expanded">
     <!-- Top banner -->
     <% if content_for?(:banner) %>
       <div class="banner">
         <%= yield :banner %>
       </div>
     <% end %>
-    <header class="header <%= 'header--interior' if request.path_info != '/' %>" data-nav-target="header">
+    <header class="header <%= 'header--interior' if request.path_info != '/' %>" data-nav-target="header collapse">
       <div class="l-wrap--header">
-        <%= link_to(root_path, title: 'RubyGems', class: 'header__logo-wrap', data: { nav_target: "headerLogo" }) do %>
+        <%= link_to(root_path, title: 'RubyGems', class: 'header__logo-wrap', data: { nav_target: "logo" }) do %>
           <span class="header__logo" data-icon="⬡">⬢</span>
           <span class="t-hidden">RubyGems</span>
         <% end %>
-        <a class="header__club-sandwich" href="#" data-nav-target="sandwichIcon">
+        <a class="header__club-sandwich" href="#" data-action="nav#toggle focusin->nav#focus click@window->nav#hide">
           <span class="t-hidden">Navigation menu</span>
         </a>
 
         <div class="header__nav-links-wrap">
-          <%= form_tag search_path, class: search_form_class, method: :get do %>
-            <%= search_field_tag :query, params[:query], placeholder: t(".header.search_gem_html"), class: "header__search", autocomplete: "off", data: { nav_target: "headerSearch" } %>
-            <ul id="suggest" class="suggest-list"></ul>
-            <%= label_tag :query do %>
-              <span class="t-hidden"><%= t(".header.search_gem_html") %></span>
+          <h1 class="t-hidden">RubyGems</h1>
```

Comment:
This breaks the tests, since there is already other `h1` on the page in `main` section. It seems `h1` is still [recommended to be used only once](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page), but the impact of not doing so is unclear to me.

Reply:
Aha! You spotted it! This is a rebase failure I think. I couldn't figure it out because I didn't remember changing this. Thank you!

Reply:
Ok, updated. I was staring at this in the code and didn't see another h1 (and I thought this was supposed to be here). Thanks for finding it.